packaging/distributions-dependencies/build.gradle.kts

```diff
@@ -29,6 +29,12 @@ val junit5Version = "5.10.5"
 val mavenVersion = "3.9.5"
 val mavenResolverVersion = "1.9.16" // Should remain in-sync with `mavenVersion`
 val nativePlatformVersion = "0.22-milestone-28"
+/**
+ * Should be in sync with:
+ * tooling API-related docs & snippets
+ * buildship: CreateBuildshipEclipseProjectTask#calculateManifest
+ * StaticLoggerProvider#REQUESTED_API_VERSION
+*/
```

Comment:
Would be nice to do it automatically, but this would require templating for snippets/untrivial logic